packages/rest/src/lib/CDN.ts

```diff
@@ -201,8 +201,13 @@ export class CDN {
 	 */
 	private makeURL(
 		route: string,
-		{ allowedExtensions = ALLOWED_EXTENSIONS, extension = 'png', size }: Readonly<MakeURLOptions> = {},
+		{ allowedExtensions = ALLOWED_EXTENSIONS, extension = 'webp', size }: Readonly<MakeURLOptions> = {},
 	): string {
+		// Stickers cannot have a `.webp` extension
+		if (route.startsWith('/stickers/')) {
+			extension = 'png';
+		}
```

Comment:
We cannot set `extension: 'json'` with this, and this code is rather troublesome since it may throw false positives or replace the format in unexpected ways, plus this is running for every single route as opposed to stickers only.